jta-and-hibernate-standalone/src/main/java/org/jboss/narayana/quickstart/jta/TransactionalConnectionProvider.java

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
IT is normal to omit copyrights from quickstarts since the intention is to encourage developers to reuse our examples.